api/models/fixtures/agent_label.json

```diff
@@ -75,6 +75,42 @@
         },
         "Yes"
       ]
+    },
+    "RegionHasCustomTopicUpdate": {
```

Comment:
> This conditional seems to suggest that it is whether or not the region allows custom topic updates

That is the case.

We have always been able to update the runtime attribute of Lambda, and already did so for things like the lifecycle handler in https://github.com/convox/rack/pull/1519/files.

We haven't been able to update the runtime attribute of CF custom handlers due to a CloudFormation limitation. That limitation of CF is now gone in some regions.

When it is gone in every region we can remove the mapping and conditional logic altogether, and hardcode 'nodejs4.3' for all lambda functions, custom topics and otherwise.